exchanges/ftx/ftx_types.go

```diff
@@ -381,7 +381,7 @@ type LTCreationData struct {
 	Token         string    `json:"token"`
 	RequestedSize float64   `json:"requestedSize"`
 	Pending       bool      `json:"pending"`
-	CreatedSize   float64   `json:"createdize"`
+	CreatedSize   float64   `json:"createdSize"`
```

Comment:
Out of scope: But if you can, Just noticed this ID should of type int64 and in the RequestTokenCreationData type below.